its/ruling/pom.xml

```diff
@@ -27,5 +27,10 @@
       <artifactId>sonar-analyzer-commons</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.sonarsource.sonar-lits-plugin</groupId>
+      <artifactId>sonar-lits-plugin</artifactId>
```

Comment:
Why this new dependency?

Reply:
@pynicolas suggested we added this dependency to debug ITs locally.

Reply:
Ah ok :+1:

Reply:
Orchestrator now looks into the local Maven repository to find the lits plugin. This new dependency forces Maven to download it before running the tests. Potential problem: it has to be the same version as used in the test.